documentation/staging/content/quickstart/install.md

```diff
@@ -60,7 +60,7 @@ $ helm install traefik-operator traefik/traefik \
 
     This Helm release deploys the operator and configures it to manage Domains in any Kubernetes namespace with the label, `weblogic-operator=enabled`. Because of the `enableClusterRoleBinding` option, the operator will have privilege in all Kubernetes namespaces. This simplifies adding and removing managed namespaces as you will only have to adjust labels on those namespaces. If you want to limit the operator's privilege to just the set of namespaces that it will manage, then remove this option, but this will mean that the operator has privilege only in the set of namespaces that match the selection strategy at the time the Helm release was installed or upgraded.
 
-    **Note:** Prior to version 3.1.0, the operator's Helm chart supported configuring only the namespaces that the operator would manage using a list of namespaces. The chart now supports specifying namespaces using a label selector, regular expression, or list. Review the available [Helm configuration values]({{< relref "/userguide/managing-operators/using-helm#domainnamespaceselectionstrategy" >}}).
+    **Note:** Prior to version 3.1.0, the operator's Helm chart supported configuring only the namespaces that the operator would manage using a list of namespaces. The chart now supports specifying namespaces using a label selector, regular expression, or list. Review the choices for selecting a domain namespace strategy in the [Namespace management]({{< relref "/userguide/managing-operators/namespace-management.md" >}}) document.
 
```

Comment:
How long do we need to keep this historical note around?

Reply:
+1 I think it can go - the fact that the Helm command is using the 'domainNameSelectionStrategy' option is enough of a hint that there are other name space selection options.

Reply:
Done.